examples/weekly-ticket-reports/main.bal

```diff
@@ -0,0 +1,65 @@
+import ballerina/io;
+import ballerina/oauth2;
+// import ballerina/http;
+import ballerinax/hubspot.crm.obj.tickets as hstickets;
+
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string refreshToken = ?;
+
+string createdTicketId = "";
+string batchTicketId1 = "";
+string batchTicketId2 = "";
+string batchTicketId3 = "";
+
+hstickets:OAuth2RefreshTokenGrantConfig auth = {
+    clientId,
+    clientSecret,
+    refreshToken,
+    credentialBearer: oauth2:POST_BODY_BEARER 
+};
+
+hstickets:ConnectionConfig config = {auth};
+final hstickets:Client TicketClient = check new hstickets:Client(config);
+
+public function main() returns error? {
+    hstickets:CollectionResponseSimplePublicObjectWithAssociationsForwardPaging ticketList = check TicketClient->/.get();
+    int totalTickets = ticketList?.results.length();
+    int lowPriorityTickets = 0;
+    int highPriorityTickets = 0;
+    int mediumPriorityTickets = 0;
+    foreach var ticket in ticketList.results {
+        if (ticket?.properties["hs_ticket_priority"] == "HIGH") {
```

Comment:
Remove the parenthesis around the `if` condition statements, which is considered an anti-pattern in Ballerina. Check similar places.

```suggestion
        if ticket?.properties["hs_ticket_priority"] == "HIGH" {
```